### PR TITLE
[T4A4][T10-B2] Li Zhiyuan

### PR DESCRIPTION
**Pull request by lzyuan168**
interface

### CODE REVIEW

**Comment by lzyuan168** [comment]
Ready for review

File: src/seedu/addressbook/ui/TextUi.java [hunk 1]
```diff
@@ -165,5 +166,16 @@ private static String getIndexedListForViewing(List<String> listItems) {
...
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
+    
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    private static String getPrintableString(Printable... printables){
+    	final StringBuilder sb = new StringBuilder();
+    	for(Printable p: printables) {
+    		sb.append(p.getPrintableString().append(" "));
```
Comment (nirandiw) [inline src/seedu/addressbook/ui/TextUi.java:176]: should be comma separated right?

File: src/seedu/addressbook/data/person/Printable.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
```
Comment (nirandiw) [inline src/seedu/addressbook/data/person/Printable.java:2]: header comments?

**Comment by nirandiw** [comment]
@lzyuan168 Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**